### ISSUE TITLE
any plans to bring back the option to use multiple threads?

### ISSUE THREAD

**Issue by svural** [brentp/somalier#64]
it would be useful.

**Comment by brentp** [comment]
i don't think so. i think with #63 then somalier will be scalable enough. multiple threads will make it at most 64X faster which is not enough when the algorithm is n^2.
